v22.2/create-external-connection.md

```diff
@@ -0,0 +1,170 @@
+
+`CREATE EXTERNAL CONNECTION` will validate the URI by writing, reading, and listing a test file to the external storage URI. If you're using a [KMS URI](take-and-restore-encrypted-backups.html), `CREATE EXTERNAL CONNECTION` will encrypt and decrypt a file. You'll find a `crdb_external_storage_location` file in your external storage as a result of this test. Each of the operations that access the external connection is aware of the raw URI that is parsed to configure, authenticate, and interact with the connection.
+
+The [privilege model](#required-privileges) for external connections means that you can delegate the creation and usage of external connections to the necessary users or roles.
+
+You can also use the following SQL statements to work with external connections:
+
+- [`SHOW CREATE EXTERNAL CONNECTION`](show-create-external-connection.html)
+- [`DROP EXTERNAL CONNECTION`](drop-external-connection.html)
+
+## Required privileges
+
+To create an external connection, a user must have the [`EXTERNALCONNECTION` system privilege](security-reference/authorization.html#supported-privileges). `root` and [`admin`](security-reference/authorization.html#admin-role) users have this system privilege by default and are capable of granting the `EXTERNALCONNECTION` privilege to other users and roles with or without the [`GRANT OPTION`](grant.html). 
+
+For example: 
+
+~~~sql
+GRANT SYSTEM EXTERNALCONNECTION TO user;
+~~~
+
+To use a specific external connection during an operation, the user must also have the `USAGE` privilege on that connection: 
+
+For example:
+
+~~~sql
+GRANT USAGE ON EXTERNAL CONNECTION backup_bucket TO user;
+~~~
+
+## Synopsis
+
+<div>
+{% remote_include https://raw.githubusercontent.com/cockroachdb/generated-diagrams/{{ page.release_info.crdb_branch_name }}/grammar_svg/create_external_connection.html %}
+</div>
+
+### Parameters
+
+Parameter | Description
+----------+-------------
+`connection_name` | The name that represents the external connection.
+`connection_URI` | The storage/sink URI that the external connection will reference.
+
+## Supported external storage and sinks
+
+Storage or sink      | Operation support               
+---------------------+---------------------------------
+[Amazon S3](use-cloud-storage-for-bulk-operations.html) | Backups, restores, imports, exports
+[Amazon S3 KMS](take-and-restore-encrypted-backups.html#aws-kms-uri-format) | Encrypted backups
+[Azure Storage](use-cloud-storage-for-bulk-operations.html) | Backups, restores, imports, exports
```

Comment:
Ahhh, a good point, but I didn't because this isn't necessarily just about the authentication where the filters on that cloud storage bulk ops come into play. I think I'll leave as-is, but I'm hoping to re-configure that page after release and so can think about how the links for reference could be used.